app/components/edit_channel_info/edit_channel_info.js

```diff
@@ -354,16 +358,18 @@ export default class EditChannelInfo extends PureComponent {
                         </View>
                     </TouchableWithoutFeedback>
                 </KeyboardAwareScrollView>
-                <KeyboardTrackingView style={style.autocompleteContainer}>
+                <View style={[style.autocompleteContainer, {bottom: autocompleteOffset}]}>
```

Comment:
if the above is answer to my question is true, maybe you should change it for

```suggestion
                const bottomStyle = Platform.select({ios: {bottom: autocompleteOffset}, android: undefined});
                <View style={[style.autocompleteContainer, bottomStyle]}>
```

not sure if applying the suggestion directly will have eslint errors, so maybe do it manually.